src/models/shape_loader.cpp

```diff
@@ -952,9 +952,18 @@ geo::ShapePtr loadShape(const std::string& model_path, tue::config::Reader cfg,
         std::string uri_path;
         if (cfg.value("uri", uri_path))
         {
+            geo::Vec3 scale(1, 1, 1);
+            std::string scale_str;
+            if (cfg.value("scale", scale_str))
+            {
+                std::vector<std::string> scale_vector = split(scale_str, ' ');
+                scale.x = std::stod(scale_vector[0]);
```

Comment:
Should protect against not getting 3 strings in the vector I think, or something will go wrong

Reply:
Another option is to use the `stod` conversion itself to read the 3 values (it has an output parameter that stores the first invalid value).